lib/miq_pglogical/connection_handling.rb

```diff
@@ -2,10 +2,27 @@ class MiqPglogical
   module ConnectionHandling
     extend ActiveSupport::Concern
 
+    included do
+      delegate :logical_replication_supported?, :to => :class
+    end
+
     module ClassMethods
+      def logical_replication_supported?
+        return @logical_replication_supported if defined?(@logical_replication_supported)
+
+        is_superuser = ActiveRecord::Base.connection.select_value("select usesuper from pg_user where usename = CURRENT_USER;", "SQL")
```

Comment:
You're killin' me smalls

```suggestion
        is_superuser = ActiveRecord::Base.connection.select_value("SELECT usesuper FROM pg_user WHERE usename = CURRENT_USER")
```

Reply:
I don't know what you're talking about... 

I clearly didn't spend more time on finding a meme than on finding all of these... 😕